### ISSUE TITLE
TOC links broken

### ISSUE THREAD

**Issue by gDelgado14** [dillonkearns/elm-pages#11]
In https://elm-pages.com/docs and https://elm-pages.com/docs/directory-structure, both table of contents links are not navigating to their respective locations on the page.

Using Firefox Quantum 69.0.3 (64-bit)

**Comment by kuituhirvi** [comment]
This likely is due to human error and not due to software failure. It seems that the HTML <h2> tag id values and accompanying <a> tag href values are case-sensitive. In plain English the second headings identification value must match 1-to-1 towards the anchor's hypertext reference value, in other words they must a complete match, not even in capitulation might they differ.

Could be that I'm mistaken, for I've not been reading up on the specifications as of late and can also confess that am currently far from familiarity with Elm neither elm-pages. It would just seem unnecessary LoC's to have capitulated & non-capitulated tags produced from the same input and that remains my reasoning for suggesting user error.

**Comment by dillonkearns** [comment]
Pushed a fix, thank you both for the info!
